### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by renzop** [iterativ/openopc2#52] [Q&A]
Use this thread for Discussions about the installation

**Comment by KellyControlsandAutomation** [comment]
I just found this software and am starting to work with it in hopes that I can use Linux as a OPC client to connect to a Windows Server machine that is running Siemens Simatic OPC Server.  Is there a way to directly connect to an OPC DA Server on a Windows machine with this software or is it mandatory to install the openopc2 service on the Windows machine?  I would rather not have to install software on the Windows system if at all possible.

**Reply by renzop** [reply]
Unfortunately this is not possible. Opc Classic (DA & AE etc.) are based on the Windows COM interface. This COM interface is Windows only. For your case i would recommend to use Openopc2 Gateway on the windows machine (run it as a Windows Service) , and then program your Custom software with Openopc2 as Python Library on the Linux System.
